.github/workflows/linux.yml

```diff
@@ -163,13 +171,26 @@ jobs:
           sudo apt update
           sudo apt install -y libxkbcommon-dev libmysqlclient21
 
+      # TODO: Figure out why this error only occurs on the Linux container when building with -DGIT_REVISION=ON
```

Comment:
Probably related to this: https://github.com/actions/runner-images/issues/6775

I think the newer git version installed by the `Install git PPA in docker container` step causes this.

Reply:
Yeah, that's rather annoying. I forget the reason, but I think I added the PPA because the `checkout` action needs a newer version of git to work.